src/popup/components/Scrollable.vue

```diff
@@ -1,11 +1,22 @@
 <template>
   <div class="scrollable">
-    <div class="scrollable-inner">
+    <div
+      class="scrollable-inner"
+      :style="{'scrollbar-width': scrollbarWidth}"
```

Comment:
CSS scrollbar-width is not supported in most of the browsers: https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-width#browser_compatibility